### PR TITLE
Update of the parameters for testing

### PR DESCRIPTION
**Pull request by patapufate**

### CODE REVIEW

File: vectornav/params/vn100.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+serial_port: /dev/ttyUSB2
 
 # Acceptable baud rates : 9600, 19200, 38400, 57600, 115200, 230400, 460800, 921600
 # Datasheet states 128000 works but from experiments it does not.
 serial_baud: 115200
 
 # Acceptable data rates in Hz: 1, 2, 4, 5, 10, 20, 25, 40, 50, 100, 200
 # Baud rate must be able to handle the data rate
-async_output_rate: 40
+async_output_rate: 25
```
Comment (hidmic) [inline vectornav/params/vn100.yaml:10]: @patapufate why these changes?

File: vectornav/src/main.cpp [hunk 2]
```diff
@@ -461,3 +309,199 @@ void BinaryAsyncMessageReceived(void* userData, Packet& p, size_t index)
...
+            ros::WallDuration(diagnosticsUpdater.getPeriod()));
+        // Need to make sure we disconnect properly. Check if all ok.
+    }
+
+    // Node has been terminated
+    vs.unregisterAsyncPacketReceivedHandler();
+    ros::Duration(0.5).sleep();
+    vs.disconnect();
+    ros::Duration(0.5).sleep();
+
```
Comment (hidmic) [inline vectornav/src/main.cpp:502]: @patapufate `run` is missing a `return` statement.

File: vectornav/src/main.cpp [hunk 3]
```diff
@@ -461,3 +309,199 @@ void BinaryAsyncMessageReceived(void* userData, Packet& p, size_t index)
...
+}
+
+int main (int argc, char *argv[]){
+    run(argc,argv);
```
Comment (hidmic) [inline vectornav/src/main.cpp:506]: @patapufate did you mean 
```suggestion
    return run(argc,argv);
```
?

File: fin_control/src/fin_control.cpp [hunk 4]
```diff
@@ -388,7 +392,7 @@ void FinControl::workerFunc() {
   while (fincontrolEnabled) {
     reportAngles();
     // sleep to maintain 25Hz update period
-    usleep(40000);  // if changing this number update durations above.
+    usleep(int(1000000*reportAngleRate));  // if changing this number update durations above.
```
[the remark this answers is not in the document]
Comment (patapufate) [reply fin_control/src/fin_control.cpp:395]: fix

File: pose_estimator/src/pose_estimator.cpp [hunk 5]
```diff
@@ -166,21 +175,24 @@ class PoseEstimatorNode  //: public LogBase
...
         "stagnation check",
-        [](const pose_estimator::CorrectedData &a, const pose_estimator::CorrectedData &b) {
-          return ((a.depth == b.depth) && (a.rpy_ang.x == b.rpy_ang.x) &&
-                  (a.rpy_ang.y == b.rpy_ang.y) && (a.rpy_ang.z == b.rpy_ang.z));
+        [poseDataSteadyBand](const pose_estimator::CorrectedData &a,
+                            const pose_estimator::CorrectedData &b) {
+          return (std::fabs((a.depth - b.depth) < poseDataSteadyBand) &&
+                  std::fabs((a.rpy_ang.x - b.rpy_ang.x) < poseDataSteadyBand) &&
+                  std::fabs((a.rpy_ang.y - b.rpy_ang.y) < poseDataSteadyBand) &&
+                  std::fabs((a.rpy_ang.z - b.rpy_ang.z) < poseDataSteadyBand));
```
[the remark this answers is not in the document]
Comment (patapufate) [reply pose_estimator/src/pose_estimator.cpp:191]: Ok

File: vectornav/params/vn100.yaml (hunk 1 above)
Comment (patapufate) [reply vectornav/params/vn100.yaml:10]: it was requested by qna

File: vectornav/src/main.cpp (hunk 2 above)
Comment (patapufate) [reply vectornav/src/main.cpp:502]: it's a void function.

File: vectornav/src/main.cpp (hunk 3 above)
Comment (patapufate) [reply vectornav/src/main.cpp:506]: fixed

File: vectornav/src/main.cpp (hunk 2 above)
Comment (hidmic) [reply vectornav/src/main.cpp:502]: Given that `run` looks like `main` implementation, I'd say it should return something.

File: vectornav/src/main.cpp [hunk 6]
```diff
@@ -461,3 +309,199 @@ void BinaryAsyncMessageReceived(void* userData, Packet& p, size_t index)
...
+    // Node has been terminated
+    vs.unregisterAsyncPacketReceivedHandler();
+    ros::Duration(0.5).sleep();
+    vs.disconnect();
+    ros::Duration(0.5).sleep();
+
+}
+
+int main (int argc, char *argv[]){
+    return(run(argc,argv));
```
Comment (hidmic) [inline vectornav/src/main.cpp:506]: @patapufate did you mean 
```suggestion
    return run(argc,argv);
```
? Why the extra parens?

**Review by hidmic — COMMENTED** [review]
Looks like this needs a rebase.

File: vectornav/src/main.cpp (hunk 2 above)
Comment (patapufate) [reply vectornav/src/main.cpp:502]: fixed

**Review by hidmic — APPROVED** [review]
